### ISSUE TITLE
Facelift for eMail recipients

### ISSUE THREAD

**Issue by FrauCsu** [tactilenews/100eyes#404]
As a contributor
I want to receive mails e.g. from #50survivors 
in a way that's easy to read.
Links should be shortened and we need line breaks:

<img width="1075" alt="Bildschirmfoto 2020-09-28 um 14 03 43" src="https://user-images.githubusercontent.com/44811680/94431207-44be2900-0195-11eb-947f-a96aaceb2a04.png">

**Comment by tillprochaska** [comment]
@FrauCsu: I think disabling all tracking options in Sendgrid should display the original, shortened links.

Regarding the line breaks: Did the original message include line breaks, e.g. after/before salutation/greetings?

**Comment by FrauCsu** [comment]
No, original message did not include line breaks (oops), because in the software I am afraid of pressing 'enter', because I do not want to send out messages that are not ready to be send. 

Will you disable all tracking options  in Sendgrid or should I do it, @tillprochaska ?

**Comment by tillprochaska** [comment]
> No, original message did not include line breaks (oops), because in the software I am afraid of pressing 'enter', because I do not want to send out messages that are not ready to be send.

Ok, that’s good to know! I think Enter won’t submit/send a message, but we might want to make that behavior more clear to the user.

> Will you disable all tracking options in Sendgrid or should I do it, @tillprochaska ?

If you’re able to, would be great if you could have a look into it as I’m a bit in a hurry today. If you don’t find the relevant options, let me know!

https://sendgrid.com/docs/ui/account-and-settings/tracking/

**Comment by FrauCsu** [comment]
<img width="944" alt="Bildschirmfoto 2020-09-28 um 14 30 33" src="https://user-images.githubusercontent.com/44811680/94432541-3709a300-0197-11eb-9fbd-b089d4463809.png">

Disable all these options? @tillprochaska

**Comment by tillprochaska** [comment]
Yes, all of them!

**Comment by FrauCsu** [comment]
Done

**Comment by tillprochaska** [comment]
Great! If you send a message (including a link) on hundred, you should see the link as you pasted it into the message form instead of those long, random strings.

**Comment by FrauCsu** [comment]
Now links will be shorter? Better readable? I cannot test just see when next mail is sent?

**Comment by FrauCsu** [comment]
Meine Mail ist nicht hinterlegt, ich kann's nicht sehen. Du?

**Comment by tillprochaska** [comment]
I just sent a test message - it works! 🎉

![image](https://user-images.githubusercontent.com/1512805/94433105-15f58200-0198-11eb-9781-52c1060eff31.png)

**Comment by FrauCsu** [comment]
Great job, well done 💪
